examples/sri-1/components/workgroup/workgroup/workgroup.ts

```diff
@@ -0,0 +1,95 @@
+        this.participants.push(participants); //[R237] A workgroup MUST consist of at least one BPI Subject participant.
+        this.securityPolicy.push(securityPolicy); //[239] A workgroup MUST have at least one security policy. Note that a security policy consists of authentication and authorization rules for the workgroup participants. Note also that one or more workgroup administrators define the workgroup security policy.
+        this.privacyPolicy.push(privacyPolicy); //[R240] A workgroup MUST have at least one security policy. A privacy policy consists of the data visibility rules for each participant.
+    }
+
+    //TODO Implement checks for actions bound by security and privacy policies
+    addParticipant(bpiSubject: BpiSubject) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions
+        return  this.participants.push(bpiSubject);
+    };
+
+    addParticipants(bpiSubjects: BpiSubject[]) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions
+        bpiSubjects.forEach(subject => {
+            this.participants.push(subject);
+        });
+    }
+
+    getParticipantById(id: string): BpiSubject {
+        return this.participants.find(subject => subject.id == id);
+    };
+
+    getAllParticipants(): BpiSubject[] {
+        return this.participants;
+    };
+
+    updateParticipant(id: string, ...update:any): BpiSubject {}; //TODO write logic for dynamic update #485
+
+    removeParticipant(id: string): BpiSubject { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions:
+        return this.participants = this.participants.filter(subject => subject.id != id);
+    };
+
+    removeParticipants(ids: string[]): BpiSubject[] { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions:
+        return this.participants = this.participants.filter(subject => !ids.includes(subject.id))
+    };
+
+    addSecurityPolicy(securityPolicy: Security) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions:
+        return this.securityPolicy.push(securityPolicy); 
+    };
+
+    removeSecurityPolicy(securityPolicy: Security) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions:
+        return this.securityPolicy = this.securityPolicy.filter(policy => policy.id != securityPolicy.id);
+    };
+
+    updateSecurityPolicy(id: string, ...updates: string[]) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions:
+        //TODO write logic for dynamic update #485
+    };
+
+    addPrivacyPolicy(privacyPolicy: Privacy) { //[R241] A workgroup administrator MUST be able to perform at minimum the following functions:
+        return this.privacyPolicy.push(privacyPolicy);
```

Comment:
I think we can keep the consistent names... this.privacyPolicies

Reply:
Please include reference in which you are basing this "consistent naming" comment.  The privacy policy for each workgroup will be a a list of, for example, data visibility rules that together compose the policy corresponding to its workgroup.

Reply:
Whether or not a workgroup should be able to maintain multiple policies is a future issue out of the scope of the PR. For the SRI, the workgroup will likely be initially capped at a single policy that includes the sum rules until expansion of that is deemed necessary for the SRI's scope of implementation.